app/assets/stylesheets/forever_style_guide/modules/_hero.scss

```diff
@@ -76,3 +56,32 @@
     @extend %text-shadow;
   }
 }
+
+/* using percentage-based padding to dynamically adjust the height (img-height / img-width * container-width) 
+http://stackoverflow.com/questions/600743/how-to-get-div-height-to-auto-adjust-to-background-size/22211990#22211990
+
+TODO: use picture element + picturefill to avoid css hacks
+ */
+.hero-no_overlap {
+  width: 100%;
+  height: 0; 
+  padding-top: 50%; /* (240 / 480 * 100) image size: 480px x 240px */
+  background-size: cover;
+  background-repeat: no-repeat;
+
+  @media (min-width: $screen-xs) {
+    padding-top: 34%; /* (484 / 1400 * 100) image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
```

Comment:
This could be made clearer and more precise using Sass' percentage instance method (http://sass-lang.com/documentation/Sass/Script/Functions.html#percentage-instance_method), something like:

```
 $height: 484px;
 $width: 1400px;

.hero-no_overlap {
  width: 100%;
  height: 0; 
  padding-top: 50%; /* (240 / 480 * 100) image size: 480px x 240px */
  background-size: cover;
  background-repeat: no-repeat;

  @media (min-width: $screen-xs) {
    padding-top: percentage($height / $width);
    //padding-top: 34%; /* (484 / 1400 * 100) image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
  }
```